docs/Heroku.md

```diff
@@ -4,7 +4,7 @@ cd stringer
 heroku create
 git push heroku master
 
-heroku config:set APP_URL=`heroku apps:info | grep -o 'http[^"]*'`
+heroku config:set APP_URL=`heroku apps:info | grep -o 'http[^"]*/$'`
```

Comment:
@Koronen thank you. GREAT
I didnt ever know `--shell` option (I needed to update heroku client)
may I send new commit with this reliable way ?